client/src/pages/code-of-conduct.js

```diff
@@ -0,0 +1,83 @@
+/* eslint-disable max-len */
+import React, { Fragment } from 'react';
+import Helmet from 'react-helmet';
+import { Grid, Col, Row } from '@freecodecamp/react-bootstrap';
+
+import { Link, Spacer } from '../components/helpers';
+
+const CodeOfConductPage = () => {
+  return (
+    <Fragment>
+      <Helmet title='Code of Conduct | freeCodeCamp.org' />
+      <Spacer />
+      <Grid className='container'>
```

Comment:
No need to set `className='container'`, it is used by default for `Grid`

Reply:
This PR is in draft. :)

All I am doing right now, is porting over the html to react. There is a fair chance that I'll miss a ton of things. I'll go through the code once again once I am ready